src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -68,41 +82,29 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
         <DialogContent>
           <Grid container spacing={2}>
             <Grid item xs={6}>
-              <Controller
-                name="projectLeadId"
-                control={control}
-                rules={{ required: true }}
-                render={({ field: { onChange, value } }) => (
-                  <>
-                    <Typography>Project Lead</Typography>
-                    <Select onChange={onChange} value={value} variant="outlined" size="small" fullWidth>
-                      {allUsers.map((p) => (
-                        <MenuItem key={p.userId} value={p.userId}>
-                          {fullNamePipe(p)}
-                        </MenuItem>
-                      ))}
-                    </Select>
-                  </>
-                )}
+              <Typography>Project Lead</Typography>
+              <NERAutocomplete
+                id="project-lead-autocomplete"
+                onChange={(_event, value) => setProjectLeadId(value?.id)}
+                options={allUsers.map((p) => ({
+                  label: fullNamePipe(p),
+                  id: p.userId.toString()
+                }))}
+                size="small"
+                placeholder="Project Lead"
               />
             </Grid>
             <Grid item xs={6}>
-              <Controller
-                name="projectManagerId"
-                control={control}
-                rules={{ required: true }}
-                render={({ field: { onChange, value } }) => (
-                  <>
-                    <Typography>Project Manager</Typography>
-                    <Select onChange={onChange} value={value} variant="outlined" size="small" fullWidth>
-                      {allUsers.map((p) => (
-                        <MenuItem key={p.userId} value={p.userId}>
-                          {fullNamePipe(p)}
-                        </MenuItem>
-                      ))}
-                    </Select>
-                  </>
-                )}
+              <Typography>Project Manager</Typography>
```

Comment:
The other two labels in this file should be form controls with form labels, can u update that while you're in here